docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -11,7 +11,7 @@ Sentry provides several configuration options to create a metric alert based on
 To create a metric alert you first need to choose a metric type. For some alert types the function is built into the alert and for others you can choose functions and parameters to apply to it. For example, if you select “Users Experiencing Errors”, that translates to the function, `count_unique(user.id)`. Since editing this function would change the nature of the alert, it is not editable and thus hidden. On the other hand, if you select “Largest Contentful Paint” the measurement used is `measurement.lcp`, you also need to choose a function, e.g. `p75()`, for a combined metric function of `p75(measurement.lcp)`.
 
 <Note>
-By default, metric alerts exclude archived issues. For instance, an alert for "Number of Errors" does not count errors from archived issues. If you want your metric alerts include archived issues, you can manually add `is:archived` to your [filters](#filters).
+By default, the "Number of Errors" metric alert filters by `is:unresolved` which excludes events from archived issues. If you want your metric alerts to include archived issues, you can manually remove `is:unresolved` from your [filters](#filters).
```

Comment:
```suggestion
When creating new metric alerts, the `is:unresolved` filter is added by default. This filter excludes events from archived and resolved issues. If you want your metric alerts to include archived issues, you can manually remove `is:unresolved` from your [filters](#filters).
```
Adjusted the wording since the filter is also applied to other metric alerts.